src/goto-symex/symex_main.cpp

```diff
@@ -136,8 +136,17 @@ void goto_symext::initialize_entry_point(
   state.top().end_of_function=limit;
   state.top().calling_location.pc=state.top().end_of_function;
   state.symex_target=&target;
-  state.dirty=util_make_unique<dirtyt>(goto_functions);
-
+  state.dirty=util_make_unique<incremental_dirtyt>();
+  if(!pc->function.empty())
```

Comment:
This should become an invariant with https://github.com/diffblue/cbmc/pull/1843

Reply:
Just making a test-gen bump for that PR